proto/ui.proto

```diff
@@ -334,6 +334,11 @@ message SavedSettings {
 	bool in_front_of_target = 13;
 	double distance_from_target = 14;
 	HealingModel healing_model = 15;
+
+	// Stop Attack Macro Info
+	bool IsUsingCrusaderStrikeStopAttack = 19;
+	bool IsUsingJudgementStopAttack = 20;
+	bool IsUsingDivineStormStopAttack = 21;
```

Comment:
Making the above change should also remove the need for these

Reply:
Hey @Adamrch can these be removed?

Reply:
removed